lib/gtfs/route.ex

```diff
@@ -0,0 +1,24 @@
+defmodule Gtfs.Route do
+  @type id :: String.t()
+
+  @type t :: %__MODULE__{
+          id: id()
+        }
+
+  @enforce_keys [
+    :id
+  ]
+
+  @derive Jason.Encoder
+
+  defstruct [
+    :id
+  ]
+
+  @spec from_csv_row(%{required(String.t()) => String.t()}) :: t()
+  def from_csv_row(row) do
```

Comment:
They are trivial now, but could it be helpful in the future to have tests for these `from_csv_row` functions?

Reply:
Hmm... I think these functions are too trivial to be worth it, but it would be good to test the data loading process, probably as a whole.
Something like
```
server = start_mocked(%{a bunch of files})
assert gtfs(server) == %__Gtfs__{all the data}
```
I'll add that into the next PR.

Reply:
Nvm. Added it to this PR.